sdk/core/core-amqp/src/requestResponseLink.ts

```diff
@@ -241,8 +247,9 @@ type NormalizedInfo = {
  *
  * Handle different variations of property names in responses emitted by EventHubs and ServiceBus.
  */
-export const getCodeDescriptionAndError = (props: any): NormalizedInfo => {
-  if (!props) props = {};
+export const getCodeDescriptionAndError = (
+  props: { [key: string]: string | number | undefined } = {}
+): NormalizedInfo => {
   return {
     statusCode: (props[Constants.statusCode] || props.statusCode) as number,
```

Comment:
should this be `??` in case the status is `0`?

Reply:
I'd be fine with changing that, though my goal with this PR was just to make the linter happy and make minimal runtime changes :)